launch/load_mh24.launch

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" ?>
 <launch>
-	<param name="robot_description" command="$(find xacro)/xacro --inorder '$(find yaskawa_mh24_support)/urdf/mh24.xacro'" />
+  <param name="robot_description" command="$(find xacro)/xacro '$(find yaskawa_mh24_support)/urdf/mh24.xacro'" />
```

Comment:
Why remove `--inorder`? As far as I understood, that is the recommended use nowadays.

Reply:
This is default starting from jade, and produces a warning starting from melodic.

Reply:
Ah I see. Leaving it out still produces a warning on kinetic/lunar I believe, but so be it. +1